notebooks/C1-build-toc-master.ipynb

```diff
@@ -58,16 +58,19 @@
     "    FULL_PREFIX = list(laplan.pcts.VALID_PCTS_PREFIX)\n",
     "    remove_prefix = [\"ENV\", \"PAR\", \"ADM\"]\n",
     "    prefix = [x for x in FULL_PREFIX if x not in remove_prefix]\n",
-    "    suffix = [\"TOC\"]\n",
     "\n",
     "    pcts = laplan.pcts.subset_pcts(\n",
     "        pcts,\n",
     "        start_date=\"2017-10-01\",\n",
     "        prefix_list=prefix,\n",
-    "        suffix_list=suffix,\n",
     "        get_dummies=True,\n",
     "    )\n",
-    "        \n",
+    "    \n",
+    "    pcts = laplan.pcts.drop_child_cases(pcts, keep_child_entitlements = True)\n",
+    "    \n",
+    "    dropme = prefix_suffix_cols = prefix + list(laplan.pcts.VALID_PCTS_SUFFIX)\n",
+    "    pcts = pcts.drop(columns = dropme)\n",
```

Comment:
Why is this necessary? Is there a problem with the exclusion of prefixes/suffixes above?

Reply:
Ahh, it's a very nuanced point. But, they wanted to know if parcels within TOC Tiers had a lot of non-TOC activity too. Like, yes, Tier 3 might have a lot of TOC entitlements, but do they also have a lot of non-TOC entitlements happening? 

Once I know which parcels fall within TOC Tiers, those parcels are such a small fraction of parcels within the city. It winnows down the non-TOC entitlements that are happening in that same period of 10/2017-onward. 

My charts depend on counting how many TOC vs non-TOC entitlements occurred. The dummies aren't important, since I want to squash all the other suffixes into non-TOC.

Reply:
Ah, got it. Thanks!